### DISCUSSION TITLE
In noticed binance using NextJs and i can't seem to find __NEXT_DATA__ in it.

### DISCUSSION THREAD

**Discussion by dexter4life** [vercel/next.js#23210] [Help]
So how do they change __NEXT_DATA__  to __APP_DATA?

**Comment by akellbl4** [comment]
It looks like they have their own fork of Next.js because they not only renamed `NEXT_DATA` into `__APP_DATA` but also, placed it almost right after `<body>` tag before all of the content of the page. They renamed the root id to `__APP`.
I think they have a lot of customizations in their fork.

**Reply by dexter4life** [reply]
yes I suspect that, but this means I can also do same right? to customize the entire nextjs package?

**Reply by akellbl4** [reply]
Yes, MIT license gives you an opportunity to make your own fork and do what you want. The only thing is it's really difficult to maintain the fork of such project.
